### PR TITLE
Update error message for operations on default identity - OSD-5379

### PR DESCRIPTION
**Pull request by csheremeta**

### CODE REVIEW

File: pkg/webhooks/identity/identity.go [hunk 1]
```diff
@@ -121,7 +121,7 @@ func (s *IdentityWebhook) authorized(request admissionctl.Request) admissionctl.
 			}
 		}
 		log.Info("Denying access", "request", request.AdmissionRequest)
-		ret = admissionctl.Denied("Prevented from accessing Red Hat managed identity provider. Please manage your identity provider through https://cloud.redhat.com/openshift")
+		ret = admissionctl.Denied("Prevented from modifying Red Hat's managed Identity. You may create/modify any Identity objects, except for 'OpenShift_SRE'.")
```
Comment (lisa) [inline pkg/webhooks/identity/identity.go:124]: Should reference the constant in this package.

**Review by lisa — CHANGES_REQUESTED** [review]
/hold

File: pkg/webhooks/identity/identity.go (hunk 1 above)
Comment (lisa) [inline pkg/webhooks/identity/identity.go:124]:
```suggestion
		ret = admissionctl.Denied(fmt.Sprintf("Prevented from modifying Red Hat's managed Identity. You may create/modify any Identity objects, except for %s.", DefaultIdentityProvider))
```

Comment (lisa) [reply pkg/webhooks/identity/identity.go:124]: Suggestion to implement for my [previous comment](https://github.com/openshift/managed-cluster-validating-webhooks/pull/116/files#r511300954).

**Review by lisa — CHANGES_REQUESTED** [review]

**Comment by csheremeta** [comment]
@lisa suggested changes have been committed, let me know if you think this looks good to go

**Comment by lisa** [comment]
/unhold

whoops, forgot one important step

**Pull request merged**